src/server/routes/meters.js

```diff
@@ -40,7 +40,7 @@ router.get('/:meter_id', async (req, res) => {
 		maxProperties: 1,
 		required: ['meter_id'],
 		properties: {
-			group_id: {
+			meter_id: {
 				type: 'number'
```

Comment:
I think this should be a string representation of an integer, not an integer, since it's from the url.